src/igvfd/schemas/changelogs/reference_file.md

```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 14
 
+* Extend `content_type` enum list to include `coding_variants`, `documentation (readme)`, `variants_coding_variants`, `variants_diseases`, `variants_diseases_genes`, `variants_phenotypes` and `variants_phenotypes_studies`.
```

Comment:
We are only including one enum per line now, so this should be divided into multiple lines